scripts/ci-entrypoint.sh

```diff
@@ -74,7 +86,7 @@ setup() {
     # this requires k8s 1.22+
     if [[ -n "${TEST_WINDOWS:-}" ]]; then
         export WINDOWS_WORKER_MACHINE_COUNT="${WINDOWS_WORKER_MACHINE_COUNT:-2}"
-        export K8S_FEATURE_GATES="WindowsHostProcessContainers=true"
+        export K8S_FEATURE_GATES="${K8S_FEATURE_GATES:-},WindowsHostProcessContainers=true"
```

Comment:
if `K8S_FEATURE_GATES` is empty, this becomes `,WindowsHostProcessContainers=true` (leading `,`) , is that intended?

Reply:
It is a mistake. Thanks. Code updated.